### PR TITLE
trek uniforms + loadout listing

### PR DESCRIPTION
**Pull request by Poojawa**
Adds in other Trek show uniforms for gear loadout use.

Needs sanity checking, because I'm p.sure I forgot where the like, snowflake point cost thing was. Also hasn't been tested in game yet, but shouldn't be too explody

Icons, of course, are done by the excellent Nienhaus.

though I did the VOY ones~

trek mob icon

### CODE REVIEW

**Comment by Nienhaus** [comment]
Those are some nice looking sprites.

**Comment by Poojawa** [comment]
Yee yee, sourced from TG's files. I edited/recolored for the VOY uniforms myself.

**Comment by Nienhaus** [comment]
I know, I made the originals :P

**Comment by Poojawa** [comment]
<3 I'll ensure it's credited properly!

**Comment by Arokha** [comment]
Thanks @Nienhaus!

File: code/modules/vore/fluffstuff/custom_clothes_vr.dm [hunk 1]
```diff
@@ -988,4 +988,72 @@ Departamental Swimsuits, for general use
...
+	icon_state = "trek_voy_command"
+	item_state = "trek_voy_command_mob"
+
+/obj/item/clothing/under/fluff/trek/engsec/voy
+	icon_state = "trek_voy_engsec"
+	item_state = "trek_voy_engsec_mob"
+
+/obj/item/clothing/under/fluff/trek/medsci/voy
+	icon_state = "trek_voy_medsci"
+	item_state = "trk_vor_medsci_mob"
```
Comment (Arokha) [inline code/modules/vore/fluffstuff/custom_clothes_vr.dm]: Probably a typo if this is supposed to be 'trek_voy_medsci_mob'

**Comment by Nienhaus** [comment]
Anytime.

File: code/modules/vore/fluffstuff/custom_clothes_vr.dm (hunk 1 above)
Comment (Poojawa) [inline code/modules/vore/fluffstuff/custom_clothes_vr.dm]: Indeed it was, thanks for catching it.

**Comment by Arokha** [comment]
Have you tried this on a test server?

**Comment by JoanRisu** [comment]
Lemme take a look at these files. I think I know what the problem is.

**Comment by Poojawa** [comment]
All confirmed working in game, yaaaaay.

Thanks Joan for clearing up the exact details <3

**Comment by JoanRisu** [comment]
We'll work on the lore for these uniforms later. Until then, dis safe to merge~

**Pull request merged**